.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,38 @@
+name: Chromatic
+# All pull requests, and
+# Workflow dispatch allows you to run this workflow manually from the Actions tab
+on:
+  pull_request:
+    paths:
+      - packages/libs/react-ui/**
+  push:
+    branches:
+      - main
```

Comment:
This means that the workflow also runs after the merge, is that intentional?

Reply:
As far as I can see, all tests run again after a merge to main. I intended to follow this convention by doing the same for these. Practically, we don't **have** to run them again.